control-plane/agents/src/bin/ha/cluster/switchover.rs

```diff
@@ -173,6 +173,7 @@ impl SwitchOverRequest {
 
     /// Attempts to republish the volume by either recreating the existing target or by
     /// creating a new target (on a different node).
+    #[tracing::instrument(level = "trace", skip(self), fields(volume.uuid = %self.volume_id), err)]
```

Comment:
Are we setting the log level to trace here? Is this needed?

Reply:
This is for the span and not trace events. I pasted an example using trace on the other PR.
But actually maybe should be info since we shouldn't have many of these and may be useful when using Jaeger.

Reply:
Yep. Info is probably fine here.